FreeRTOS/FreeRTOS
# [Draft] CBMC proofs for +TCP

Draft PR for addition of +TCP CBMC proofs to this repo

Description
-----------
This PR aims to bring all the +TCP CBMC proofs in this repository. The end goal is to execute the proofs with every new PR to check whether the PR introduces any breaking change.
This is a PR that will involve quite a bit of editing and changes to allow CBMC to execute with the different directory structure of this repo (when comparing with amazon-freertos). I will make changes to it and people can review it and suggest changes while I am doing so.
Once everything is finalised, I will create a 'squashed' PR.

Test Steps
-----------
Once the files are edited to work with this repository,
- Run `python3 prepare.py` in the /FreeRTOS/Test/CBMC/proofs directory
- Commit the changes caused be prepare.py
- Run 'make' in each proof subdirectory


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
